app/src/main/java/com/work/restaurant/network/api/KakaoApi.kt

```diff
@@ -1,14 +1,17 @@
 package com.work.restaurant.network.api
 
-import com.work.restaurant.network.model.KakaoResponse
+import com.work.restaurant.network.model.kakao.KakaoResponse
 import retrofit2.Call
 import retrofit2.http.GET
 import retrofit2.http.Headers
 import retrofit2.http.Query
 
 interface KakaoApi {
     @Headers("Authorization: KakaoAK 785b8c0fb5d7046009351ac6fe2fed8b")
-    @GET("v2/local/search/keyword.json?y=37.514322572335935&x=127.06283102249932&radius=20000")
-    fun keywordSearch(@Query("keyword") keyword: String): Call<KakaoResponse>
-
+    @GET("v2/local/search/keyword.json?radius=20000&sort=accuracy")
+    fun keywordSearch(
+        @Query("query") keyword: String,
+        @Query("x") x: Double,
+        @Query("y") y: Double
```

Comment:
```suggestion
        @Query("y") y: Double
        @Query("radius") radius: Int = 20000,
        @Query("sort") sort: String = accuracy
```
이런식으로 만들어두는게 변경사항에 대응하기 좋지않을까요?

Reply:
aafde81 수정하였습니다.